internal/configconverter/config_server_test.go

```diff
@@ -244,9 +244,9 @@ func TestSimpleRedact(t *testing.T) {
 func waitForRequiredPort(t *testing.T, port string) {
 	// Wait for a relatively long time for the port to be available, given that other package tests might be using it.
 	const waitTime = 60 * time.Second
-	require.Eventually(t, func() bool {
-		return isPortAvailable(port)
-	}, waitTime, 500*time.Millisecond)
+	require.EventuallyWithT(t, func(c *assert.CollectT) {
```

Comment:
Same change as https://github.com/signalfx/splunk-otel-collector/pull/6657, just in case original usage was causing any flakiness.